### PR TITLE
[qr-filter-qml-plugin] Offer explicit result clearing. Contributes to JB#49377

### PR DESCRIPTION
**Pull request by pvuorela**
@HandsomeJaack

### CODE REVIEW

File: plugin/qrfilter.cpp [hunk 1]
```diff
@@ -10,12 +10,6 @@
 QrFilter::QrFilter(QObject *parent)
     : QAbstractVideoFilter(parent)
 {
-    connect(this, &QrFilter::activeChanged, this, [=]{
-        if (!isActive()) {
-            m_result.clear();
-            emit resultChanged(m_result);
-        }
-    });
```
Comment (HandsomeJaack) [inline plugin/qrfilter.cpp:18]: It is necessary to remove auto clearing on changing active state?

Comment (pvuorela) [reply plugin/qrfilter.cpp:18]: Not necessary but started thinking it might be a good idea to give app the control, and the same behavior is easy to do.

Comment (pvuorela) [reply plugin/qrfilter.cpp:18]: One line on the camera.

Actually, ponder if the app side should deactivate the detection when it gets a result. If the filter cleared the result automatically that could get more difficult.

**Comment by HandsomeJaack** [comment]
LGTM

**Review by HandsomeJaack — APPROVED** [review]

**Pull request merged**